extensions/wikia/Piggyback/Piggyback_body.php

```diff
@@ -115,9 +113,10 @@ function successfulLogin() {
 
 		$cu = User::newFromName( $this->mUsername );
 
-		if ( !$cu->checkPassword( $this->mPassword ) &&
-				!HeliosUser::checkAuthenticationStatus( $cu->getName(), $this->mPassword, WikiaResponse::RESPONSE_CODE_FORBIDDEN )) {
-
+		$authResult = $cu->checkPassword( $this->mPassword );
+		if ( !$authResult->success() &&
+			$authResult->checkStatus( WikiaResponse::RESPONSE_CODE_FORBIDDEN )
+		) {
 			if ( $retval = '' == $this->mPassword ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$retval" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)